package.json

```diff
@@ -28,15 +28,15 @@
     "cors": "2.8.5",
     "express": "4.17.1",
     "express-jwt": "^6.0.0",
-    "express-pino-logger": "^6.0.0",
     "helmet": "4.6.0",
     "http-errors": "^1.8.0",
     "ioredis": "^4.27.1",
     "ioredis-mock": "^5.5.5",
     "jsonwebtoken": "^8.5.1",
     "node-fetch": "^2.6.1",
-    "pino": "^6.11.3",
-    "pino-colada": "^2.1.0"
+    "pino": "^7.6.4",
```

Comment:
We have a weird mix of pinned and `^` versions in this file.  In a follow-up, we should standardize (probably on pinned like Telescope).  cc @DukeManh.